specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-05-01-preview/apimpolicystatus.json

```diff
@@ -0,0 +1,85 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ApiManagementClient",
+    "description": "REST API for getting policy status in an Azure API Management deployment.",
+    "version": "2023-05-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/policyStatus": {
```

Comment:
>policyStatus [](http://example.com/codeflow?start=128&length=12)

You are missing a point GET, there are other services that could require it, see our reference doc on [inconsistent crud](https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html)